### PR TITLE
Convert PageContainerLongPressHandler to Kotlin

### PR DESCRIPTION
**Pull request by cooltey**

### CODE REVIEW

File: app/src/main/java/org/wikipedia/page/PageContainerLongPressHandler.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package org.wikipedia.page
+
+import org.wikipedia.Constants
+import org.wikipedia.LongPressHandler.WebViewMenuCallback
+import org.wikipedia.history.HistoryEntry
+import org.wikipedia.readinglist.database.ReadingListPage
+
+class PageContainerLongPressHandler(private val fragment: PageFragment) : WebViewMenuCallback {
+    override fun onOpenLink(entry: HistoryEntry) {
```
Comment (sharvaniharan) [inline app/src/main/java/org/wikipedia/page/PageContainerLongPressHandler.kt:9]: Can we have an empty line here?

File: app/src/main/java/org/wikipedia/LongPressHandler.kt [hunk 2]
```diff
@@ -20,7 +20,7 @@ import org.wikipedia.util.UriUtil.isValidPageLink
 
 class LongPressHandler(view: View, private val historySource: Int, private val callback: LongPressMenu.Callback) : OnCreateContextMenuListener, OnTouchListener {
     interface WebViewMenuCallback : LongPressMenu.Callback {
```
Comment (sharvaniharan) [inline app/src/main/java/org/wikipedia/LongPressHandler.kt:22]: Empty line?

File: app/src/main/java/org/wikipedia/page/PageContainerLongPressHandler.kt (hunk 1 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/page/PageContainerLongPressHandler.kt:10]: Done!

File: app/src/main/java/org/wikipedia/LongPressHandler.kt [hunk 3]
```diff
@@ -20,7 +20,7 @@ import org.wikipedia.util.UriUtil.isValidPageLink
...
-        val wikiSite: WikiSite
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/LongPressHandler.kt:23]: Why can't we keep `wikiSite` nonnull?

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/LongPressHandler.kt (hunk 3 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/LongPressHandler.kt:23]: The `override val wikiSite = fragment.title?.wikiSite` in `PageContainerLongPressHandler` has a chance to be a `null` if the `fragment.title` returns `null`.

We can do either put `!!` after `wikiSite` or change the callback to become a nonnull type of variable.

**Review by sharvaniharan — APPROVED** [review]

**Pull request merged**